pkgs/by-name/co/corrscope/package.nix

```diff
@@ -1,49 +1,39 @@
 { lib
-, mkDerivationWith
 , python3Packages
 , fetchFromGitHub
-, fetchpatch
-, wrapQtAppsHook
-, ffmpeg
-, qtbase
+, ffmpeg-full
+, libsForQt5
 , testers
 , corrscope
 }:
 
-mkDerivationWith python3Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "corrscope";
-  version = "0.8.1";
+  version = "0.9.0";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "corrscope";
     repo = "corrscope";
     rev = version;
-    hash = "sha256-pS7upOYZAjgR3lWxny8TNZEj3Rrbg+L90ANZWFO9UPQ=";
+    hash = "sha256-kOPhVm4epIhBSsgQVKNCoQ7DZcMG/b3sapxwwKo/V+U=";
   };
 
-  patches = [
-    # https://github.com/corrscope/corrscope/pull/446
-    (fetchpatch {
-      name = "remove-setuptools-dependency.patch";
-      url = "https://github.com/corrscope/corrscope/commit/70b123173a7a012d9f29d6d3a8960b85caf6cc79.patch";
-      hash = "sha256-YCtb7v8cGP0pdceAKeoempnRzw+LRKQqDb3AfN0z/9s=";
-    })
-  ];
-
   pythonRelaxDeps = [ "attrs" "ruamel.yaml" ];
 
-  nativeBuildInputs = [
+  nativeBuildInputs = (with libsForQt5; [
     wrapQtAppsHook
-  ] ++ (with python3Packages; [
+  ]) ++ (with python3Packages; [
     poetry-core
     pythonRelaxDepsHook
   ]);
 
   buildInputs = [
-    ffmpeg
+    ffmpeg-full
```

Comment:
Shouldn't only ffmpeg be used and not full? What are we missing from the normal variant?

Reply:
https://github.com/NixOS/nixpkgs/issues/271863#issuecomment-1837484140

> `corrscope`: Uses `ffplay` for export preview, which depends on `withFullDeps`.

Reply:
see #296128

Reply:
Switched to regular `ffmpeg` now, and tested that it works fine.